sphinx/builders/htmlhelp.py

```diff
@@ -168,6 +169,18 @@
     'zh_TW': (0x404, 'cp950'),
 }
 
```

Comment:
Could you add 2 blank lines before a function and classes?
flake8 warns here and before `HTMLHelpBuilder` class.
https://travis-ci.org/sphinx-doc/sphinx/jobs/471503849#L520-L521